packages/app/src/app/overmind/effects/vscode/index.ts

```diff
@@ -219,17 +219,48 @@ export class VSCodeEffect {
 
     // Only set the read only state when the editor is initialized.
     this.initialized.then(() => {
-      // ReadOnly mode is derivative, it's based on a couple conditions, of which the
-      // most important one is Live. If you're in a classroom live session as spectator,
-      // you should not be allowed to edit.
+      // ReadOnly mode is derivative, it's based on a couple conditions. The most important
+      // one is the `freePlanEditingRestricted` flag. The second most important one is live.
+      // If you're in a classroom live session as spectator, you should not be allowed to edit.
```

Comment:
I changed this comment with the `freePlanEditingRestricted` change I made, but the "old" part confuses me a bit. It talks about `live` being the most important, but it seems with the implementation that when `state.editor.currentSandbox.git` exists, that's fine too. Does this mean that when it's a sandbox connected to a git repository the user is (was) always able to edit? If so I'll change the comment to reflect this.

Hope maybe @christianalfoni or @CompuIves can shed a little light on this :)